HTMLToQPDF/Components/ParagraphComponent.cs

```diff
@@ -171,5 +172,42 @@ private TextStyle ParseStyleString(string styleString, TextStyle textStyle)
             }
             return textStyle;
         }
+
+        private string GetAlignmentAttribute(HtmlNode element)
+        {
+            foreach (var attr in element.Attributes)
+            {
+                var value = attr.Value;
+                if (attr.Name == "class")
+                {
+                    List<string> classes = new List<string>(value.Split(' '));
+                    foreach (string className in classes)
```

Comment:
why not `foreach (string className in value.Split(' '))` You have unnecessary allocations and list creations. Same on line 195

Reply:
updated in latest commit